R/method-correlation.R

```diff
@@ -301,12 +66,16 @@ setMethod("selfCorrelation", signature("AbundanceData"), function(data, method =
   data <- pruneFeatures(data, predicateFactory('sd', stdDevThreshold), verbose)
 
   abundances <- getAbundances(data, FALSE, FALSE, verbose)
-  corrResult <- correlation(abundances, method=method, verbose=verbose)
+  corrResult <- correlation(abundances, method = method, format = 'data.table', verbose = verbose)
```

Comment:
bc i want more control over which objects get passed to the buildComputeResult thingy, so that the metadata for the data1 and data2 objects dont just say 'unknown'